### DISCUSSION TITLE
Browser Support for Syncing All Bookmarks At Root Level?

### DISCUSSION THREAD

**Discussion by balthanon** [floccusaddon/floccus#1523] [Q&A] [answered]
So I am working on getting Floccus set up to replace Eversync, which has never really worked quite right, but was stable enough that I could at least force it manually.  I am using Floccus to sync every browser I have installed on all of my computers (don't really care about Android as I don't really use bookmarks on my phone anyway.)

My initial test case was Chrome and Brave (which I don't use often, so didn't care about) and I am syncing the root folder because I want the bookmarks bar and the bookmark menu/other bookmarks/bookmarks sidebar to be synced up as well.  I had a few issues with it not syncing properly, but after clearing everything out and redoing it, it appeared to be working as I would expect.  Adding in Vivaldi and Edge, Edge seems to be working okay, Vivaldi just dumps everything but the bookmark bars into the Trash folder.  Firefox worked even worse than that in some respects, where it created a new folder for all the Floccus folders when I tried to set it to the root plus apparently duplicated the entire set of bookmarks in other bookmarks once or twice.

So I suppose my question at this point is... is root syncing across browsers actually supported, or should I have one profile for the bookmark bar and another for the other bookmarks/bookmarks menu stuff?  And if I do that, is there any issue with having it all synced via Google Drive with the same account, just using a different xbel file?

If it is supported, should I put an issue out in Github for the issues, or is there something I'm likely missing to get this working properly?

**Comment by marcelklehr** [comment, marked as answer]
> is root syncing across browsers actually supported

Yes, I try to support this, but there may be bugs on the odd browser, I do not test this regularly. Everything beyond Firefox and Chrome is basically a gamble if it will work.

>  is there any issue with having it all synced via Google Drive with the same account, just using a different xbel file?

not at all, you can have multiple profiles syncing via the same GDrive with different files. That's the whole idea of profiles :)

**Reply by balthanon** [reply]
Would you like me to create a bug for the problems I'm having in the other browsers for tracking purposes?  If not, I'll just go ahead and set up multiple profiles instead and mark the answer.  It's probably safer in the long run anyway if I ever go even further afield with newer browsers and it sounded like you were busy with your day job.

**Reply by marcelklehr** [reply]
Feel free to open issues for the problems! :)
